### ISSUE TITLE
lex_utils: needs update

### ISSUE THREAD

**Issue by sagotch** [geneweb/geneweb-contrib#8]
I am pretty sure that https://github.com/geneweb/geneweb-contrib/blob/master/lex/lex_utils.ml#L148 need to be updated in order to make the script work.

```
  let repo_src = Filename.concat repo "src" in
```

We should be able to pass multiple directories to be looked up (recursively).

**Comment by hgouraud** [comment]
Yes, it should be "lib" indeed. I had corrected it, but somehow it came back!!
The current code reads recursively folders in "lib".
What needs to be done is reading as well into plugins, but we ned to be a bit more explicit about which source file contains which message missing in the lexicon.

**Comment by sagotch** [comment]
> Yes, it should be "lib" indeed. I had corrected it, but somehow it came back!!

Probably my fault then, I had troubles rebasing the PR.

> What needs to be done is reading templates into plugins/xx/assets, and this might be done with:

And lexicon coming with plugins as well.

**Comment by hgouraud** [comment]
Could not get lex_utils.ml to work properly.
Restarted from old version and created lex_utils-2.ml which seems to work now (reads plugins lexicons).
Coding style can be improved!!

**Comment by sagotch** [comment]
> Could not get lex_utils.ml to work properly. (lack of line numbers in error messages does not help!!)

I need to work on this, missing location is definitively a problem.
